src/main/java/wolox/training/model/Book.java

```diff
@@ -0,0 +1,125 @@
+package wolox.training.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private Integer pages;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    public Book() {
+
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
I would remove this setter for now as it doesn't seem likely that anyone will use it given the id get's generated automatically.

Reply:
Yes, you're right!
Done!